### PR TITLE
(CT-117) Implement puppet-db export

### PR DESCRIPTION
**Pull request by GabrielNagy**
Still needs tests, however I am unsure whether I've taken the correct approach with this.

Depends on #103

### CODE REVIEW

File: cmd/puppet-db/export.go [hunk 1]
```diff
@@ -1,21 +1,60 @@
...
+	Run: executeExportCommand,
 }
 
 func init() {
 	rootCmd.AddCommand(exportCmd)
 }
 
 func executeExportCommand(cmd *cobra.Command, args []string) {
-	fmt.Print("not implemented")
+	anonymizationProfile := viper.GetString("anon")
```
Comment (ciprianbadescu) [inline cmd/puppet-db/export.go:32]: Should we check that value is one of: full, moderate, low ?

Comment (GabrielNagy) [reply cmd/puppet-db/export.go:32]: I also thought of that, would be another thing to maintain if it changes frequently. I'll add a check though, it doesn't seem likely that these profiles would change often.

**Comment by ciprianbadescu** [comment]
Compared export with orig/new puppetdb, the only change is expected ad is backup timestamp => good to merge.

File: cmd/puppet-db/export.go (hunk 1 above)
Comment (ciprianbadescu) [reply cmd/puppet-db/export.go:32]: On second thought the allowed values may not be known in advance, better send it transparently to server

**Pull request merged**